lib/commands/web-view/set-web-view.ts

```diff
@@ -0,0 +1,47 @@
+///<reference path="../../.d.ts"/>
+"use strict";
+import commandParams = require("../../common/command-params");
+
+export class SetWebViewCommand implements ICommand {
+	constructor(private $webViewService: IWebViewService,
+		private $injector: IInjector,
+		private $errors: IErrors) { }
+	
+	public execute(args: string[]): IFuture<void> {
+		return this.$webViewService.enableWebView(args[0], args[1]);
+	}
+	
+	public canExecute(args: string[]): IFuture<boolean> {
+		return (() => {
+			if(!args[0] || !args[1]) {
+				this.$errors.fail(`Platform and web view name are mandory params.`);
+			}
+			
+			this.$webViewService.validateFrameworkVersion();
+			
+			var supportedWebViews = this.$webViewService.supportedWebViews;
+			
+			// Validate platform
+			var platform = args[0].toLowerCase();
+			var platforms = _.keys(supportedWebViews);
+			if(!_.contains(platforms, platform)) {
+				this.$errors.failWithoutHelp(`Invalid platform. The valid platforms that support web view are: ${platforms.join(", ")}`);
+			}
+			
+			// Validate webView
+			var webView = args[1].toLowerCase()
+			var webViewNames = this.$webViewService.getWebViewNames(platform);
+			if(!_.contains(webViewNames, webView)) {
+				this.$errors.failWithoutHelp(`Invalid web view. The valid webViews for ${platform} platform are: ${webViewNames.join(", ")}`);
+			}
+			
+			return true;
+			
+		}).future<boolean>()();
+	}
+	
+	public allowedParameters: ICommandParameter[] = [
+		this.$injector.resolve(commandParams.StringCommandParameter), 
```

Comment:
as you are implementing canExecute method, you do not need this. It is enough to have:
```TypeScript
public allowedParameters: ICommandParameter[] = []
```